examples/resources/maas_network_interface_tag/resource.tf

```diff
@@ -0,0 +1,27 @@
+resource "maas_network_interface_tag" "test" {
+  machine      = "abc123"
+  interface_id = "12"
```

Comment:
based on your implementation this is a number. What about removing the double quotas from every `interface_id`?